sagemaker-training-compiler/huggingface/pytorch_single_gpu_single_node/vision-transformer/vision-transformer.ipynb

```diff
@@ -0,0 +1,324 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.        max_run=60 * 60 * 24,  # 60 minutes
</pre><p>Actually limit this to 60 minutes instead of 24 hours</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3541/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>